frontend/src/components/svgs/index.js

```diff
@@ -203,6 +203,22 @@ const InstaIcon = ({ className = "scale-100" }) => (
   </svg>
 );
 
+const Circle = ({ color }) => (
+  <svg
+    width="80"
+    height="80"
+    viewBox="0 0 80 80"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
```

Comment:
This could be done different with tailwind:
```suggestion
    className="w-20 h-20 fill-none"
    viewBox="0 0 80 80"

```